hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/util/BackupUtils.java

```diff
@@ -594,8 +598,10 @@ public int compare(BackupInfo o1, BackupInfo o2) {
       }
 
       private long getTimestamp(String backupId) {
-        String[] split = backupId.split("_");
-        return Long.parseLong(split[1]);
+        Iterator<String> i = Splitter.on('_').split(backupId).iterator();
```

Comment:
Use Iterators.get

Reply:
I see this as recurring feedback, got it. Will apply this pattern to these things. There are other cases in other modules where the same change is appropriate.